code/datums/martial/boxing.dm

```diff
@@ -35,7 +35,7 @@
 
 	D.apply_damage(damage, STAMINA, affecting, armor_block)
 	log_combat(A, D, "punched (boxing) ")
-	if(D.getStaminaLoss() > 50)
+	if(D.getStaminaLoss() > 50 && D.mind.martial_art && D.mind.martial_art == /datum/martial_art/boxing)
```

Comment:
Not everyone has a mind. This runtimes when hitting monky man :monkey:

Reply:
good catch